jre_emul/android/platform/libcore/luni/src/test/java/libcore/java/util/zip/ZipFileTest.java

```diff
@@ -32,14 +32,15 @@
 import java.util.zip.ZipFile;
 import java.util.zip.ZipOutputStream;
 
+
 public final class ZipFileTest extends AbstractZipFileTest {
 
     @Override
     protected ZipOutputStream createZipOutputStream(OutputStream wrapped) {
         return new ZipOutputStream(wrapped);
     }
 
-    /* J2ObjC removed: do not support android.system.OsConstants. */
+
     // http://b/30407219
```

Comment:
Should this file also be checked in along with the java.util.logging package changes?

Reply:
I'm okay with including it even though it isn't related, since the change is small and cosmetic. Hopefully I'm not teaching Mary any bad habits, though. :-)

Reply:
Not a problem. This looks good to me so far, but I'll defer to you whether this is ready. Thanks!

Reply:
Thanks for reminding! I remember trying not to commit this file, but I may accidentally add it. Just reverted the change.